server/src/main/resources/db/migration/V2_0__Object_Permissions.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS object_permissions (
+    id INTEGER PRIMARY KEY,
```

Comment:
I think ints can get up to like a few billion values, which _should_ be fine, but if we start storing arbitrary data for external companies this could maybe end up being something we'll have to deal with in the future.  Maybe we should swap to like UUID or BigInt instead?